barretenberg/cpp/src/barretenberg/protogalaxy/combiner.test.cpp

```diff
@@ -53,7 +53,7 @@ TEST(Protogalaxy, CombinerOn2Instances)
             ProverInstances instances{ instance_data };
             instances.alphas.fill(bb::Univariate<FF, 12>(FF(0))); // focus on the arithmetic relation only
             auto pow_polynomial = PowPolynomial(std::vector<FF>{ 2 });
-            auto result = prover.compute_combiner(instances, pow_polynomial);
+            auto result = prover.compute_combiner</*disable_optimisation=*/true>(instances, pow_polynomial);
```

Comment:
i think it would be great to have these tests run for the optimised version as well

Reply:
but they actually shouldn't be modified if I understand correctly, just some of them skipped

Reply:
Updated the values